### ISSUE TITLE
releases contain the bundle vendor resulting in tens of MB instead of a few KB

### ISSUE THREAD

**Issue by jsvd** [Icinga/logstash-output-icinga#7]
looking at rubygems.org we can see gems are very large:

1.2.0 - December 28, 2017 (44.6 MB)
1.1.0 - October 05, 2017 (77.6 MB)
1.0.0 - May 10, 2017 (32.4 MB)

and after unpacking the 1.2.0 we can see the vendor directory is big:
```
/tmp/logstash-output-icinga-1.2.0 % du -sh *
4.0K	CHANGELOG.md
4.0K	CONTRIBUTORS
4.0K	Gemfile
4.0K	LICENSE
4.0K	README.md
 20K	lib
4.0K	logstash-output-icinga.gemspec
 12K	spec
 69M	vendor
```

This is because the gemspec includes everything in the vendor directory and the gem was built after a bundle install was done using `--path ./vendor`

**Comment by bobapple** [comment]
@jsvd thank you for the hint. I had to bump the version number to correct this.

I would like to close this repository here and work on the logstash-plugins/logstash-output-icinga repo. It seems like this plugin is not being published to the official logstash output plugins website. Is there anything I can do?

**Comment by bobapple** [comment]
This was fixed in v1.2.1
